packages/toolkit/src/lib/vdp-sdk/pipeline/actions.ts

```diff
@@ -173,7 +171,6 @@ export async function triggerUserPipelineReleaseAction({
       payload,
       {
         headers: {
-          "instill-return-traces": returnTraces ? "true" : "false",
```

Comment:
@orangecoloured What is the reason removing this field? We need this to gain the traces

Reply:
The code does not use that prop.
```
"Access-Control-Allow-Headers": "instill-return-traces, instill-share-code, Instill-Requester-Uid",
```
It always adds that header.

Reply:
Oh... yes you are right, nvm, all these action is revamped in the sdk PR